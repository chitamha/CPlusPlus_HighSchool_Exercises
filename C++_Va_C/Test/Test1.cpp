#include <bits/stdc++.h>
#define fi first
#define se second
#define llll pair<ll, ll>
#define ll long long
#define maxn 1000006
using namespace std;

int main(){
    //freopen("text1.inp", "r", stdin);
    //freopen("text1.out", "w", stdout);
    ios_base::sync_with_stdio(false);
    cin.tie(nullptr); cout.tie(nullptr);

    return 0;
}
