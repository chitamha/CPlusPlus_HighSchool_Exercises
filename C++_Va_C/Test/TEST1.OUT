7 1 6 2
3 2 4 4
5 2 5 5
5 2 6 6
5 3 6 8
4 4 6 10
11